/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
Traceback (most recent call last):
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/requests/compat.py", line 11, in <module>
    import chardet
ModuleNotFoundError: No module named 'chardet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/sion/Desktop/youtubeCommunityPostAlarm/youtube_community_alarm.py", line 4, in <module>
    from youtube_community import YoutubeCommunity, REGEX
  File "/Users/sion/Desktop/youtubeCommunityPostAlarm/youtube_community.py", line 1, in <module>
    import requests
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/requests/__init__.py", line 45, in <module>
    from .exceptions import RequestsDependencyWarning
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/requests/exceptions.py", line 9, in <module>
    from .compat import JSONDecodeError as CompatJSONDecodeError
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/requests/compat.py", line 13, in <module>
    import charset_normalizer as chardet
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/__init__.py", line 24, in <module>
    from .api import from_bytes, from_fp, from_path, is_binary
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/api.py", line 5, in <module>
    from .cd import (
  File "/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/cd.py", line 14, in <module>
    from .md import is_suspiciously_successive_range
ImportError: dlopen(/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/md.cpython-39-darwin.so, 0x0002): tried: '/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/md.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/md.cpython-39-darwin.so' (no such file), '/Users/sion/Desktop/data/youtubeCommunityPostAlarm/lib/python3.9/site-packages/charset_normalizer/md.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64'))
